httpclient5-cache/src/main/java/org/apache/hc/client5/http/impl/cache/ResponseProtocolCompliance.java

```diff
@@ -101,8 +100,8 @@ private void warningsWithNonMatchingWarnDatesAreRemoved(
         boolean modified = false;
         for(final Header h : warningHeaders) {
             for(final WarningValue wv : WarningValue.getWarningValues(h)) {
-                final Date warnDate = wv.getWarnDate();
-                if (warnDate == null || warnDate.equals(responseDate)) {
+                final Instant warnInstant = wv.getWarnInstant();
+                if (warnInstant == null || warnInstant.equals(responseDate)) {
                     newWarningHeaders.add(new BasicHeader(HeaderConstants.WARNING,wv.toString()));
```

Comment:
A bit confusing, `getWarnDate()` has been renamed, but `responseDate` not?

Reply:
Hi @michael-o 
surely you have. reason, but the idea is to do it in stages.
TY

Reply:
OK

Reply:
Thank you @michael-o 
I just want avoid mistake and mess the code.   My idea is make the apis change progressively. 
TY